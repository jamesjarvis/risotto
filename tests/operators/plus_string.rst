println("a" + "b")

println(1 + "b")
println("a" + 1)

println(1.0 + "b")
println("a" + 1.0)

println(true + "b")
println("a" + true)

println(false + "b")
println("a" + false)

s1 := "s1"
s1 = s1 + "s1"
println(s1)

s2 := "s2"
s2 = s2 + 1
println(s2)

s3 := "s3"
s3 = s3 + 1.0
println(s3)

s4 := "s4"
s4 = s4 + true
println(s4)

s5 := "s5"
s5 = s5 + false
println(s5)
