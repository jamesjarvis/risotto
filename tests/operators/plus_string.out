ab
1b
a1
1.000000b
a1.000000
trueb
atrue
falseb
afalse
s1s1
s21
s31.000000
s4true
s5false
